Makefile

```diff
@@ -270,3 +273,39 @@ upload-review-backup: read-deployment-config read-tf-config # make review upload
 
 backup-review-database: read-deployment-config # make review backup-review-database APP_NAME=1234
 	bin/backup-review-database ${POSTGRES_DATABASE_NAME} ${paas_env}
+
+deploy-domain-resources: check-auto-approve domain-azure-resources # make apply deploy-domain-resources AUTO_APPROVE=1
```

Comment:
should read "make register deploy-domain-resources AUTO_APPROVE=1"
more references below that need to change, plus it looks like the register block is missing